### ISSUE TITLE
Executing CQL3(Cassandra.Client.execute_cql3_query) queries regardless to the CQL version in the config

### ISSUE THREAD

**Issue by OleksandrSemenov** [Impetus/kundera#280]
Kundera version: 2.5
Cassandra version: 1.2.5

Hi
Kundera executes cql3 queries regardless to the version number which is set in the configuration.
Bug located in com.impetus.client.cassandra.CassandraClientBase in executeCQLQuery:

``` java
protected CqlResult executeCQLQuery(String cqlQuery) throws InvalidRequestException, UnavailableException,
            TimedOutException, SchemaDisagreementException, TException
    {
        Cassandra.Client conn = null;
        Object pooledConnection = null;
        pooledConnection = getPooledConection(persistenceUnit);
        conn = getConnection(pooledConnection);
        conn.set_cql_version(getCqlVersion());
        try
        {
            return conn.execute_cql3_query(ByteBufferUtil.bytes(cqlQuery),
                    org.apache.cassandra.thrift.Compression.NONE, consistencyLevel);
        }
        finally
        {
            releaseConnection(pooledConnection);
        }
    }
```

Here cql version is set by conn.set_cql_version, but query is always executed by method for CQL3

Thank you for your work & best regards,
Oleksandr

**Comment by mevivs** [comment]
KK,
Please look into this.

-Vivek

**Comment by mevivs** [comment]
Fixed and releasing with 2.6. Please verify.
-Vivek
